air-quality-ui/src/services/forecast-data-service.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { DateTime } from 'luxon'
+
+import { getForecastData } from './forecast-data-service'
+
+describe('Forecast Data Service', () => {
+  beforeEach(() => {
+    jest.spyOn(global, 'fetch').mockImplementationOnce(() =>
+      Promise.resolve({
+        json: () => Promise.resolve([]),
+      } as unknown as Response),
+    )
+  })
+  it('should getForecastData for city by default', async () => {
```

Comment:
Could add a test where we supply a different location type?

Reply:
Because I've typed it I actually can't without a type error! I could force it by using an unknown type maybe?